block_rich_text.go

```diff
@@ -92,6 +95,78 @@ func (u RichTextUnknown) RichTextElementType() RichTextElementType {
 	return u.Type
 }
 
+type RichTextListElementType string
+
+const (
+	RTEListOrdered RichTextListElementType = "ordered"
+	RTEListBullet  RichTextListElementType = "bullet"
+)
+
+type RichTextList struct {
+	Type     RichTextElementType     `json:"type"`
+	Elements []RichTextElement       `json:"elements"`
+	Style    RichTextListElementType `json:"style"`
```

Comment:
```suggestion
	Style    RichTextListElementType `json:"style"`
	Indent   int                     `json:"indent"`
```

Hello 👋 Wondering if you can also add support for the `indent` field. For some reason, I can't find official documentation on this field, but here is an [example with the Block kit builder](https://app.slack.com/block-kit-builder/#%7B%22blocks%22:%5B%7B%22type%22:%22rich_text%22,%22elements%22:%5B%7B%22type%22:%22rich_text_list%22,%22style%22:%22bullet%22,%22indent%22:0,%22elements%22:%5B%7B%22type%22:%22rich_text_section%22,%22elements%22:%5B%7B%22type%22:%22text%22,%22text%22:%22Hello,%20I'm%20a%20bullet%20point!.%22%7D%5D%7D%5D%7D,%7B%22type%22:%22rich_text_list%22,%22style%22:%22bullet%22,%22indent%22:1,%22elements%22:%5B%7B%22type%22:%22rich_text_section%22,%22elements%22:%5B%7B%22type%22:%22text%22,%22text%22:%22Indented%20item.%22%7D%5D%7D%5D%7D%5D%7D%5D%7D).

Suggested changes should be sufficient to support it. 🌆